<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomeDialog</class>
 <widget class="QDialog" name="welcomeDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>405</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>wikiLYNX Startup</string>
  </property>
  <widget class="QTextBrowser" name="appLogo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/base/images/wikiLYNX_logo.png&quot; width=&quot;100&quot; height=&quot;60&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="openLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="privateLogo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#161616&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/base/images/DAWN.png&quot; width=&quot;100&quot; height=&quot;50&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
   <property name="openLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Welcome to wikiLYNX!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier Prime</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>wikiLYNX</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>511</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabPlay">
    <attribute name="title">
     <string>Play</string>
    </attribute>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
      </font>
     </property>
     <property name="text">
      <string>Level: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="passcodeInput">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Select Level</string>
     </property>
    </widget>
    <widget class="QPushButton" name="initButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>INIT!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Difficulty: </string>
     </property>
    </widget>
    <widget class="QLabel" name="difficulty">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Checkpoints: </string>
     </property>
    </widget>
    <widget class="QLabel" name="chk">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../assets/assets.qrc">
       <normaloff>:/base/images/refresh.png</normaloff>:/base/images/refresh.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSettings">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="allowSitesToggle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow other sites</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="killToggle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow users to change game window </string>
     </property>
    </widget>
    <widget class="QPushButton" name="cLogsButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>140</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Logs</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sLogsButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>140</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Logs</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editLevelButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Levels</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
